### PR TITLE
[DOC] Clarify the difference of paddle.norm and np.linalg.norm

### PR DESCRIPTION
**Pull request by ZHUI**
### PR types

Others
### PR changes

Docs
### Describe

Clarify the difference of paddle.norm and np.linalg.norm

### CODE REVIEW

**Comment by ZHUI** [comment]
文档预览
image

image

File: python/paddle/tensor/linalg.py [hunk 1]
```diff
@@ -177,11 +177,17 @@ def norm(x, p='fro', axis=None, keepdim=False, name=None):
     Returns the matrix norm (Frobenius) or vector norm (the 1-norm, the Euclidean
     or 2-norm, and in general the p-norm for p > 0) of a given tensor.
 
+    .. note::
+        This norm API is different from `np.linalg.norm`.
+        This api supports high-order input tensors (rank >= 3), and certain axes need to be pointed out to calculate the norm.
```
Comment (TCChenlong) [inline python/paddle/tensor/linalg.py]: and certain axes ->  and certain axis

Comment (ZHUI) [reply python/paddle/tensor/linalg.py]: done

**Review by TCChenlong — APPROVED** [review]
LGTM

**Pull request merged**